frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -1546,25 +1546,12 @@
 
         read-only?      (mf/use-ctx ctx/workspace-read-only?)
 
-        add-color
-        (mf/use-fn
-         (fn [value _]
-           (st/emit! (dwl/add-color value))))
-
         add-color-clicked
         (mf/use-fn
-         (mf/deps file-id)
-         (fn [event]
-           (st/emit! (dw/set-assets-section-open file-id :colors true)
-                     (ptk/event ::ev/event {::ev/name "add-asset-to-library"
-                                            :asset-type "color"}))
-           (modal/show! :colorpicker
-                        {:x (.-clientX event)
-                         :y (.-clientY event)
-                         :on-accept add-color
-                         :data {:color "#406280"
-                                :opacity 1}
-                         :position :right})))
+          (fn [event]
+            (let [pos-x (.-clientX event)
+                  pos-y (.-clientY event)]
+              (st/emit! (dc/select-color pos-x pos-y)))))
```

Comment:
Prefer use dom/get-client-position (it returns point); so pass a point instance to select-color instead of x and y